config.go

```diff
@@ -54,17 +54,17 @@ const (
 	defaultAllowHighFees       = false
 
 	// ticket buyer options
-	defaultMaxFee            dcrutil.Amount = 1e8
+	defaultMaxFee            dcrutil.Amount = 1e7
 	defaultMinFee            dcrutil.Amount = 1e6
 	defaultMaxPriceScale                    = 2.0
 	defaultMinPriceScale                    = 0.7
 	defaultAvgVWAPPriceDelta                = 2880
 	defaultMaxPerBlock                      = 5
 	defaultBlocksToAvg                      = 11
-	defaultFeeTargetScaling                 = 1.05
+	defaultFeeTargetScaling                 = 1.0
```

Comment:
Ok, yah that's reasonable, especially since this is the first step towards fixing the problem generally.